ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/InMemoryStorageProvider.java

```diff
@@ -38,15 +39,19 @@
 public class InMemoryStorageProvider implements StorageProvider {
 
   public static MutableBlockchain createInMemoryBlockchain(final Block genesisBlock) {
-    return createInMemoryBlockchain(genesisBlock, new MainnetBlockHeaderFunctions());
+    return createInMemoryBlockchain(
+        genesisBlock, MainnetProtocolSchedule.DEFAULT, new MainnetBlockHeaderFunctions());
```

Comment:
The protocol schedule should be an argument, not a presumed default.  We will want to have test cases on both sides of the typed transactions fork.